src/Dfe.PlanTech.Application/Submissions/Queries/GetSubmissionStatusesQuery.cs

```diff
@@ -18,14 +18,12 @@ public GetSubmissionStatusesQuery(IPlanTechDbContext db, IUser userHelper)
         _userHelper = userHelper;
 
     }
-
-    public Task<List<SectionStatusDto>> GetSectionSubmissionStatuses(IEnumerable<ISectionComponent> sections)
+    
+    public Task<List<SectionStatusDto>> GetSectionSubmissionStatuses(string categoryId)
     {
         int establishmentId = _userHelper.GetEstablishmentId().Result;
```

Comment:
Would you mind making this `await`ed instead of `.Result` please whilst you're at it? Been bothering me for ages but never had an excuse to fix it